app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -85,6 +81,27 @@ def id_claim_key
             required_variable_names: [TOKEN_APP_PROPERTY_VARIABLE]
           )[TOKEN_APP_PROPERTY_VARIABLE]
         end
+
+        def id_claim_value
+          return @id_claim_value if @id_claim_value
+
+          @id_claim_value = @jwt_authenticator_input.decoded_token.dig(
+            *@parse_claim_path.(claim: id_claim_key)
```

Comment:
i think we are using` .call` instead of just `.`, please validate me

Reply:
Also what do you think to extract this `*@parse_claim_path.(claim: id_claim_key)` to function `parsed_claim_path`

Reply:
See the [conversation](https://github.com/cyberark/conjur/pull/2396#discussion_r737196656) in the PR introduced the class.

Reply:
Regarding `.` vs `.call` have note found any reference.
@tzheleznyak may be you know

Reply:
I prefer to use .call or .func_name because its more readable .

Reply:
Will add explicit `.call`...